apps/rush/src/start.ts

```diff
@@ -90,7 +90,7 @@ const launchOptions: rushLib.ILaunchOptions = { isManaged, alreadyReportedNodeTo
 
 // If we're inside a repo folder, and it's requesting a different version, then use the RushVersionManager to
 // install it
-if (rushVersionToLoad && rushVersionToLoad !== currentPackageVersion) {
+if ( RushCommandSelector._getCommandName() !== 'myrush' && (rushVersionToLoad && rushVersionToLoad !== currentPackageVersion)) {
```

Comment:
```suggestion
if (RushCommandSelector._getCommandName() !== 'myrush' && (rushVersionToLoad && rushVersionToLoad !== currentPackageVersion)) {
```

Reply:
Can you add the suggestion to my updated changes? I should have pulled in your suggestions before I pushed new commits.